apps/functions-subscription/src/adapters/azure/cosmosdb/__tests__/trial.test.ts

```diff
@@ -84,6 +85,48 @@ describe('makeTrialsCosmosContainer', () => {
       expect(mockDB.container('').item).toBeCalledWith(id, ownerId);
     });
   });
+  describe('list', () => {
+    it('should return list of trials', async () => {
+      const mockDB = makeDatabaseMock();
+
+      const anotherTrial = {
+        ...aTrial,
+        id: 'anotherTrialId012345678901' as TrialId,
+        name: 'anotherTrialName' as NonEmptyString,
+        description: 'anotherTrialDescription',
+      };
+
+      const trials = [aTrial, anotherTrial];
+
+      mockDB.container('').items.query.mockReturnValueOnce({
+        fetchAll: () => Promise.resolve({ resources: trials }),
+      });
+
+      const actual = await makeTrialsCosmosContainer(
+        mockDB as unknown as Database,
+        containerName,
+      ).list({ pageSize: 25, maximumId: undefined, minimumId: undefined })();
+
+      expect(actual).toStrictEqual(E.right(trials));
+      expect(mockDB.container('').items.query).toHaveBeenCalledTimes(1);
+    });
+
+    it('should return an empty list', async () => {
+      const mockDB = makeDatabaseMock();
+
+      mockDB.container('').items.query.mockReturnValueOnce({
+        fetchAll: () => Promise.resolve({ resources: [] }),
+      });
+
+      const actual = await makeTrialsCosmosContainer(
+        mockDB as unknown as Database,
+        containerName,
+      ).list({ pageSize: 25, maximumId: undefined, minimumId: undefined })();
```

Comment:
suggestion: as already said in another comment, the ideal would be:

```suggestion
      ).list({ pageSize: 25 })();
```

Reply:
Resolved in eff99bbaf41355bc1a98ee131ab9a4d2694136f4